Tutorials/Inverese_Kinematics_with_IMUs/imuInverseKinematics_Setup.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<OpenSimDocument Version="40000">
+	<IMUInverseKinematicsTool>
+		<!--Name of the directory where results are written. Be default this is the directory in which the setup file is be  executed.-->
```

Comment:
(noting here, but this is a core issue): Second sentence needs to be cleaned up. Should it be: "By default this is the directory from where the tool is run" or "By default this is the directory of the setup file"?

Reply:
Agreed on the text update to "By default this is the directory of the setup file" in a core PR